docker/tinode/Dockerfile

```diff
@@ -95,9 +92,6 @@ ENV TNPG_AUTH_TOKEN=
 # Tinode Push Gateway user name.
 ENV TNPG_USER=
 
-# Enable Tinode Push Gateway request payload compression.
```

Comment:
Can you remove https://github.com/tinode/chat/blob/devel/docker/tinode/config.template#L114 as well please?

Reply:
done